fxdc/config.py

```
@@ -157,5 +340,78 @@ def get_class_name(self, class_: type) -> str:
+        with open(file, "r") as f:
+            data = f.read()
+        if not data.startswith("!CONFIG FILE!"):
+            raise NoConfigFound("Invalid config file format")
+        
+        data:dict[str, dict] = loads(data).original
+        for classname, meta_data in data.items():
+            classname = classname[7:]
+            if classname not in self.custom_classes_names:
+                raise ClassNotLoaded(
+                    f"Class {classname} is not loaded in the config"
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)